src/main/resources/insert_service_test_data.sql

```diff
@@ -0,0 +1,3 @@
+insert into public.service
```

Comment:
I suppose this should be in integration resource folder

Reply:
done